Tools/i18n/pygettext.py

```diff
@@ -5,7 +5,7 @@
 # Minimally patched to make it even more xgettext compatible
 # by Peter Funk <pf@artcom-gmbh.de>
 #
-# 2002-11-22 J�rgen Hermann <jh@web.de>
+# 2002-11-22 Jürgen Hermann <jh@web.de>
```

Comment:
I assume that the change here is a re-encoding from Latin-1 to UTF-8.  If that is so, please remove the encoding line at the top of the file.

Reply:
I think it is better to not include any unrelated changes. If you want to change the encoding of the file, it should be done in a separate issue. But non-default encoding may be a part of testing.

Reply:
Indeed it was a change of encoding, thanks for pointing that out; my text editor seems to save files in UTF-8 by default. I've reverted to encoding to ISO 8859-1.